### ISSUE TITLE
Request to publish extension to open-vsx.org

### ISSUE THREAD

**Issue by ar-daniel** [smcpeak/vscode-default-keys-windows#4]
Please publish this extension to the open-vsx.org so that it can be used in open source builds of vsCode. ( like vscodium )

Instructions wiki for publishing
https://github.com/eclipse/openvsx/wiki/Publishing-Extensions
main ref source
https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions-marketplace

open issue here
https://github.com/microsoft/vscode/issues/31168

Thank you

**Comment by smcpeak** [comment]
It's interesting and unfortunate that Microsoft has seemingly chosen to divide the VSCode community by disallowing marketplace access from forks.  I've followed the linked instructions and the result is here:

https://open-vsx.org/extension/smcpeak/default-keys-windows

Let me know if this works for you.

(At the moment, that site says my namespace is public.  I've submitted a [claim](https://github.com/EclipseFdn/open-vsx.org/issues/245) for ownership.)
